### DISCUSSION TITLE
View recipe calories/nutrition

### DISCUSSION THREAD

**Discussion by pbozeman** [TandoorRecipes/recipes#2847] [General]
I have nutrition properties set on the foods for a recipe, but don't see how to get it summarized on the recipe page.  Is there a way to do that, or is it still TBD?

And, from there, is there a way to view nutrition stats of a meal plan?

**Comment by smilerz** [comment]
Neither recipe rollup or meal
Plan are currently possible.

**Comment by vabene1111** [comment]
the nutritions should show underneath the recipe, meal plan and shopping will follow in the future

**Reply by pbozeman** [reply]
@vabene1111, I'm not seeing any nutrition in my self hosted version.  I just imported one of my recipes into your demo version and set the properties for a few ingredients.  I don't see nutrition info there either.  see: https://app.tandoor.dev/view/recipe/47325.

Is there something else I need to do?

**Reply by pbozeman** [reply]
I did some more debugging, and I'm on the right track now.  My units did not have "Base Unit" set.     
 def calculate_recipe_properties(self, recipe) wasn't able to do the necessary conversions.  The PropertyViewComponent was getting all 0s for fp.total_value, so it wasn't displaying anything.
 
To make it work, I had to set the base unit for grams, pounds, etc.  I'm not sure if these were default units or were auto-created at some point, but either way, it would probably be helpful to have these get defaults of their Base Units.

**Reply by vabene1111** [reply]
yes I agree, that is not very straight forward, I need to look into this if there is any solution to get this fixed in a user friendly way.
